test/e2e/specs/toppanel.spec.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'Basic top panel test': function (client) {
```

Comment:
Make sure all the tests you added are named consistently with each other and with existing tests